pkg/kubectl/cmd/config/config.go

```diff
@@ -66,59 +80,323 @@ func NewCmdConfig(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	return cmd
 }
 
-func (o *pathOptions) getStartingConfig() (*clientcmdapi.Config, string, error) {
-	filename := ""
+func NewDefaultPathOptions() *PathOptions {
+	ret := &PathOptions{
+		LocalFile:  ".kubeconfig",
+		GlobalFile: os.Getenv("HOME") + "/.kube/.kubeconfig",
```

Comment:
Can we change this to `$HOME/.kube/kubeconfig`? The parent folder (`.kube`) is hidden, so there's no real reason to hide it.